realsense2_camera_msgs/CMakeLists.txt

```diff
@@ -28,16 +28,18 @@ find_package(ament_cmake REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
```

Comment:
Why do we need to add this?

Reply:
the colcon build fails for this package without this line
since we are using sensor_msgs/Image and sensor_msg/CameraInfo types inside the RGBD.msg

Reply:
OK